client/src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { ReactComponent as LogoIcon } from '../image/HeaderLogo.svg';
+
+export default function ShowLogo() {
```

Comment:
consider to name your components and files in the same way :) it makes it easier to track your components!

Reply:
fixed :)